ckan/logic/action/get.py

```diff
@@ -1172,7 +1172,7 @@ def resource_file_metadata_show(context, data_dict):
     if not resource:
         raise NotFound
 
-    _check_access('resource_show', resource_context, data_dict)
+    _check_access('resource_file_metadata_show', resource_context, data_dict)
```

Comment:
Why not retain the existing access check? `resource_show` seems like a suitable permission for this action.

Reply:
in auth it is just a wrapper
```
def resource_file_metadata_show(context, data_dict):
    return authz.is_authorized('resource_show', context, data_dict)

```

Reply:
tried with only resource_show in testing and it did not like that, seems it needs a matching auth name for tests to work. so yes it is needed even tho its a wrapper. there is other instances like this in the codebase.